modules/serverless-build-func-kn.adoc

```diff
@@ -8,29 +8,50 @@
 
 Before you can run a function, you must build the function project by using the `kn func build` command. This command creates an OCI container image that can be run locally on your computer or on an {product-title} cluster. The build command uses the function project name and the image registry name to construct a fully qualified image name for your function.
 
-By default, `kn func build` creates a container image using the Red Hat Source-to-Image (S2I) technology. To use the link:https://buildpacks.io/[CNCF Cloud Native Buildpacks] technology instead, supply the `-b` flag:
+[id="serverless-build-func-kn-image-containers_{context}"]
+== Image container types
 
-.Example build command
+By default, `kn func build` creates a container image by using Red Hat Source-to-Image (S2I) technology.
+
+.Example build command using Red Hat Source-to-Image (S2I)
+[source,terminal]
+----
+$ kn func build
+----
+
+You can use link:https://buildpacks.io/[CNCF Cloud Native Buildpacks] technology instead, by adding the `-b` flag to the command:
```

Comment:
Do you think it would help the reader understand and remember if we use the full tag names in help text? `--build`

Reply:
Yes, I actually thought about this but didn't want to change stuff that was out of scope - I think it's better to write the full tag names though, so I'll update them for this module.